packages/core/src/lib/handler/sms.handler.ts

```diff
@@ -1,11 +1,19 @@
 import { compileTemplate } from '../content/content.engine';
 import { ISmsProvider } from '../provider/provider.interface';
-import { IMessage, ITriggerPayload } from '../template/template.interface';
+import {
+  ChannelTypeEnum,
+  IMessage,
+  ITriggerPayload,
+} from '../template/template.interface';
 
 export class SmsHandler {
   constructor(private message: IMessage, private provider: ISmsProvider) {}
 
   async send(data: ITriggerPayload) {
+    data.$attachments = data.$attachments?.filter((item) =>
```

Comment:
```suggestion
    const attachments = data.$attachments?.filter((item) =>
```
I'd prefer if we don't override function params and instead create new const for that here